_posts/2019-10-31-treadmill-desk.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Treadmill Desk
+date: 2019-10-31 16:28:05.000000000 +01:00
+type: post
+published: true
+status: publish
+categories: [Tech]
+image:
+image2:
+author: Valentino Urbano
+---
+
+A few days ago I've written about how using a [standing desk][1] is good for your health and advised everyone to do it.
+
+What I haven't mentioned is, that's not the whole story. We also have a treadmill desk at the office, and as for the standing desk it is rarely used. Which means that I can basically use it whenever I want.
+
+You should get familiar with the standing desk first and be comfortable working on it before trying the treadmill desk.
+
+## Benefits
+
+I never thought I could work while walking on it, but it goes rather slowly and for most things, once you get the hang of it, it is totally fine. You're not going to be more productive than if you're not walking, but the loss of productivity is minimal and the health benefits are huge.
```

Comment:
You're not going to be more productive than if you're not walking, but the loss of productivity is minimal and the health benefits are huge. - > The loss of productivity is minimal and the health benefits are huge.

Reply:
Like that sounds that is only good. Sounds also be pointed out that the loss is there.